iam-policy-a.json

```diff
@@ -0,0 +1,205 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "acm1",
+            "Effect": "Allow",
+            "Action": "acm:DescribeCertificate",
+            "Resource": "arn:aws:acm:*:*:certificate/*"
+        },
+        {
+            "Sid": "acm2",
+            "Effect": "Allow",
+            "Action": [
+                "acm:ListCertificates",
+                "acm:ListTagsForCertificate"
+            ],
+            "Resource": "*"
+        },
```

Comment:
Is this used anymore now that we do it in K8s?

Reply:
Hm, this just made me realize a major flaw with this overall approach (that of the internal deployer PR, that is): It doesn't account for any aws calls our deployer make. It's only testing the majority of aws calls that are performed by terraform, but not the ones the deployer itself will do.